rocketc/rocketc.py

```diff
@@ -176,10 +191,16 @@ def user_data(self):
         user_data = {}
         user_data["email"] = user.emails[0]
         user_data["role"] = runtime.get_user_role()
+        user_data["course_id"] = runtime.course_id
         user_data["course"] = re.sub('[^A-Za-z0-9]+', '', runtime.course_id._to_string()) # pylint: disable=protected-access
         user_data["username"] = user.opt_attrs['edx-platform.username']
         user_data["anonymous_student_id"] = runtime.anonymous_student_id
-        user_data["default_group"] = self.default_channel
+
+        if self.channel == "Team Discussion":
+            user_data["default_group"] = self.team_channel
```

Comment:
is there any difference between default_group and default_channel?

Reply:
Ok as I see, the default group can be the default channel if Team Discussion is not activated